assets/mmbot_config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "cex_infos": {
+    "binance": {
+      "cex_endpoint": "https://api.binance.com",
+      "cex_api_public_key": "<your public key here>",
+      "cex_api_private_key": "<your private key here>"
+    },
+    "coinbase": {
+      "cex_endpoint": "https://api.pro.coinbase.com",
+      "cex_api_public_key": "<your public key here>",
+      "cex_api_private_key": ""
+    }
+  },
+  "price_infos": {
+    "coinpaprika": {
+      "endpoint": "https://api.coinpaprika.com/v1"
```

Comment:
should be `price_endpoint` if it follows same naming convention

Reply:
There will be more informations than endpoint in a latter stage no ?

Reply:
naming convention should be applied globally regardless of if we add more config params

Reply:
Perfect !